libs/designer/src/lib/core/actions/bjsworkflow/settings.ts

```diff
@@ -190,7 +195,8 @@ export const getOperationSettings = (
     },
     invokerConnection: {
       isSupported: isInvokerConnectionSupported(isTrigger, nodeType, rootNodeId),
-      // value: TODO in next PR
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Shouldn't need to do this. `invokerConnection` should allow `connectionReferences: ConnectionReferences | undefined`, then the `!` can be removed here.